### DISCUSSION TITLE
Logo, renk, isim değişiklikleri gibi modifikasyonları nasıl yapabiliriz ?

### DISCUSSION THREAD

**Discussion by ERNCNR** [realsuayip/django-sozluk#46] [General]
Logo, renk, isim değişiklikleri gibi modifikasyonları nasıl yapabiliriz ?

**Comment by realsuayip** [comment]
HTML/CSS dosyaları ile oynayarak yapabilirsiniz. Site ismini admin panelinden site kısmından değiştirebilirsiniz.
